.github/workflows/packages.yml

```diff
@@ -93,6 +93,7 @@ jobs:
       matrix:
         arch: [x86_64, i686, ppc64le, aarch64]
         pyver: [cp36, cp37, cp38, cp39, cp310]
+        platform: [manylinux, musllinux]
```

Comment:
Honestly don't know what to call this. I picked "platform" from [PEP 656](https://www.python.org/dev/peps/pep-0656/) 🤷‍♂️

Reply:
Seems ok, and it's a local name anyway. If we find a better naming it will be easy to fix :+1: